### PR TITLE
feat: add sticky

### PR DESCRIPTION
**Pull request by lvhaibin**

### CODE REVIEW

File: components/sticky/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+</div>
+```
+
+## API
+
+### Props
+
+| Attribute | Description | Type | Default |
+| --- | --- | --- | --- |
+| offsetTop `v2.8.7` | Offset top, supports `px` `vw` `rem` unit, default `px` | _number \| string_ | `0` |
```
Comment (zangxd) [inline components/sticky/README.md:54]: delete version

File: components/sticky/README.zh-CN.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+```js
+const container = React.useRef(null);
+const [containerRef, setContainerRef] = React.useState(null);
+React.useEffect(() => {
+    setContainerRef(container.current);
+}, [container.current]);
+```
+
+```html
+<div ref={container} style={{ height: 150, backgroundColor: '#fff' }}>
```
Comment (zangxd) [inline components/sticky/README.zh-CN.md:49]: style 能去掉吗？用Box 也行啊，写 demo 要想一下 trillion 同学的怎么用

File: components/sticky/index.tsx [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+import { createNS, isDef } from '../utils';
+import { getScrollTop, getElementTop, getScroller } from '../utils/dom/scroll';
+import { View } from './style';
+
+const [bem] = createNS('sticky');
+
+interface Props {
+    zIndex?: number | string;
+    offsetTop?: number | string;
+    container?: any;
```
Comment (zangxd) [inline components/sticky/index.tsx:13]: any？

File: components/sticky/index.tsx [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+    useEffect(() => {
+        containerRef.current = container;
+    }, [container]);
+
+    const currentStyle = useMemo(() => {
+        if (!fixed) {
+            return {};
+        }
+
+        const style: Record<string, any> = {};
```
Comment (zangxd) [inline components/sticky/index.tsx:40]: any

File: components/sticky/README.zh-CN.md (hunk 2 above)
Comment (lvhaibin) [reply components/sticky/README.zh-CN.md]: done

File: components/sticky/index.tsx (hunk 3 above)
Comment (lvhaibin) [reply components/sticky/index.tsx]: done

File: components/sticky/index.tsx (hunk 4 above)
Comment (lvhaibin) [reply components/sticky/index.tsx]: done

**Pull request merged**